com.vscodium.codium.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: com.vscodium.codium
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: codium
+desktop-file-name-suffix: ' (Flatpak)'
```

Comment:
I understand you want to differentiate it but I don't think it's the way to go.

Reply:
ohh no! I miss that change, let me fix that.

Reply:
> Have you contacted upstream team about publishing their application on Flathub?

Yes, @stripedpajamas is the developer of VSCodium 

> Maintainer of VSCodium here-- I don't have capacity to maintain the flatpak version myself, but would be happy to help @noonsleeper etc whenever/however.